src/controllers/user.controller.js

```diff
@@ -0,0 +1,78 @@
+const userService = require('../services/user.service.js');
+
+const get = (_req, res) => {
+  res.sendStatus(200);
+  res.send(userService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
```

Comment:
Setting `res.statusCode` directly is not the recommended way to set the status code. Instead, use `res.status(201)` to set the status code and chain it with `send(newUser)`.